prisma/migrations/20241104165143_add/migration.sql

```diff
@@ -0,0 +1,31 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `name` on the `user` table. All the data in the column will be lost.
+  - You are about to drop the `post` table. If the table is not empty, all the data it contains will be lost.
+  - Added the required column `password` to the `User` table without a default value. This is not possible if the table is not empty.
```

Comment:
**issue**: Adding a required `password` column to the `User` table without a default value can cause issues if the table is not empty. Consider how existing records will be handled.